api_client.go

```diff
@@ -560,18 +562,22 @@ func (c *APIClient) prepareRequest(
 		// Basic HTTP Authentication
 		if auth, ok := ctx.Value(ContextBasicAuth).(BasicAuth); ok {
 			localVarRequest.SetBasicAuth(auth.UserName, auth.Password)
+			log.Printf("[DEBUG] basic auth: %s - %s", auth.UserName, auth.Password)
```

Comment:
We should not reveal sensitive info in logs, same below

Reply:
remove the debug message.